apps/www/public/rss.xml

```diff
@@ -5,9 +5,16 @@
       <link>https://supabase.com</link>
       <description>Latest news from Supabase</description>
       <language>en</language>
-      <lastBuildDate>Wed, 07 Sep 2022 16:04:00 +0000</lastBuildDate>
+      <lastBuildDate>Sun, 02 Oct 2022 16:04:00 +0000</lastBuildDate>
       <atom:link href="https://supabase.com/rss.xml" rel="self" type="application/rss+xml"/>
       <item>
+  <guid>https://supabase.com/blog/postgres-wasm</guid>
+  <title>Postgres WASM by Snaplet and Supabase</title>
+  <link>https://supabase.com/blog/postgres-wasm</link>
+  <description>We're open sourcing postgres-wasm, a PostgresQL server that runs inside a browser, with our friends at Snaplet.</description>
+  <pubDate>Sun, 02 Oct 2022 16:04:00 +0000</pubDate>
+</item>
+<item>
```

Comment:
Not sure if it's GitHub or not, but is the indentation strange here?

Reply:
it's auto generated file

Reply:
it should be indented though yea, maybe should run prettier afterwards